.github/workflows/build.yml

```
@@ -119,8 +119,44 @@ jobs:
                     (cd tests/Application && bin/console doctrine:schema:create --env=test_with_postgres)
                     composer test-with-postgres
 
+            -
+                name: Run lint container without Doctrine
+                run: |
+                    composer remove --dev doctrine/*
+                    (cd tests/Application && bin/console lint:container)    
+
             -
                 name: Run lint container without maker bundle
                 run: |
                     APP_ENV=test_without_maker composer remove --dev symfony/maker-bundle
                     (cd tests/Application && bin/console lint:container --env=test_without_maker)
+
+    tests_with_skeleton:
+        runs-on: ubuntu-latest
+        name: "Tests with the Symfony Skeleton PHP ${{ matrix.php }}${{ matrix.skeleton != '' && format(', Skeleton {0}', matrix.skeleton) || '' }}"
+        strategy:
+            fail-fast: false
+            matrix:
+                php: ["8.3"]
+                skeleton: ["^5", "^6", "^7"]
+
+        steps:
+            -
+                uses: actions/checkout@v2
```

```suggestion
                uses: actions/checkout@v4
```

There is another `v2 -> v4` at line 37